### PR TITLE
LPS-156768 Create Stubs Testray Build - Administrator

### PR DESCRIPTION
**Pull request by thaiscabrals**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayBuildAdministrator.testcase [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+	test AdminCanSortCompilationBuildName {
+
+		// TODO LPS-156585 AdminCanSortCompilationBuildName pending implementation
+
+	}
+
+	@description = "LPS-156588 - Verify if as an Administrator I should be able to sort the Builds Index"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test AdminCanSortCompilationBuildProductVersion {
```
Comment (barrosdennys) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayBuildAdministrator.testcase:112]: Is there a reason why we are adding this "Compilation" word?

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayBuildAdministrator.testcase [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+		// TODO LPS-156588 AdminCanSortCompilationBuildProductVersion pending implementation
+
+	}
+
+	@description = "LPS-156586 - Verify if as an Administrator I should be able to sort the Builds Index"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test AdminCanSortCompilationBuildPromoted {
```
Comment (barrosdennys) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayBuildAdministrator.testcase:121]: Is there a reason why we are adding this "Compilation" word?

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayBuildAdministrator.testcase [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+	test AdminCanPromoteBuild {
+
+		// TODO LPS-156583 AdminCanPromoteBuild pending implementation
+
+	}
+
+	@description = "LPS-156585 - Verify if as an Administrator I should be able to sort the Builds Index"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test AdminCanSortCompilationBuildName {
```
Comment (barrosdennys) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayBuildAdministrator.testcase:103]: Is there a reason why we are adding this "Compilation" word? Couldn't it be something like **AdminCanSortBuildByName**?

**Comment by thaiscabrals** [comment]
ci:closed

**Pull request closed without merging**